docker-compose.yml

```diff
@@ -42,6 +42,7 @@ services:
     volumes:
       - ./web/impact:/wwwroot
       - ../django-accelerator:/packages/src/django-accelerator
+      - ../New-Mentor-Directory/dist:/wwwroot/static/dist
```

Comment:
Since we're creating this special mount point just for the directory static files, we can give it a more expressive name, e.g. `/wwwroot/static/mentor-directory`.  Just a thought, not a blocker.

Reply:
There is a piece of this setup that is missing. Our current build/release process looks like the following:

- travis kicks off a build of the docker-compose.travis.yml file
- built images are pushed to ECR
- a deploy of accelerate kicks off a deploy of the built image to the ECS cluster

since there is no reference to the new mentor directory repo in travis yml file, the pushed impact api image would have none of the compiled assets for it. I just recently created a PR that adds a Dockerfile to the new mentor directory so that we can reference it in the docker-compose file:
https://github.com/masschallenge/New-Mentor-Directory/pull/18

Reply:
this is what the entry in the docker-compose.yml and docker-compose.travis.yml should look like once the aforementioned PR is merged:

```
  mentor_directory:
    build: ../New-Mentor-Directory
    volumes:
      - ../New-Mentor-Directory/dist:/usr/src/app/dist
```

the "web" docker-compose service should also contain a "depends_on" entry for "mentor_directory" so that the resources get built before mounting into the web service.